packages/runtime/client/rest/src/withdrawal/endpoints/collection.ts

```diff
@@ -78,10 +70,10 @@ export const PostResponse = t.type({
     });
 
 export const postViaAxios:(axiosInstance: AxiosInstance) => POST
```

Comment:
once again this sounds weird... should this be `makeWithdrawl` or similar?

Reply:
it's weird on purpose :-)